test/integration/pagination_test.go

```diff
@@ -50,7 +51,7 @@ func TestPagination(t *testing.T) {
 		retryErr := retry.Do(func() error {
 			_, getErr := clientset.CoreV1().Namespaces().Get(context.Background(), namespaceName, metav1.GetOptions{})
 			if !errs.IsNotFound(getErr) {
-				return errors.New("Waiting for namespace "+namespaceName+" to be deleted")
+				return errors.New("Waiting for namespace " + namespaceName + " to be deleted")
```

Comment:
This is a white space only change and unnecessary.

Reply:
this is the result of running `goimports` on the file after saving the changes I made elsewhere

Reply:
Maybe we don't normally run `goimports` on test code?

Reply:
it's possible. I know this isn't the first time I've edited a file and the formatter has made changes to lines that I didn't edit and you comment on it. I do know that the integration test are a little weird though. I have to disable the go plugin for vim when editing the the integration tests because it doesn't like that the package they are in is called `main`. It might break plugins in other editors too but I have no clue

Reply:
Thanks for the tip, that fixed the gopls errors I would get in the integration tests